db/migrate/20221011111744_drop_categories_table.rb

```diff
@@ -0,0 +1,9 @@
+class DropCategoriesTable < ActiveRecord::Migration[7.0]
```

Comment:
Why you are dropping categories table?

Reply:
deleted the migration and added new migration to drop 'introduction' from videos table